src/js/common/schemaform/review/SubmitButtons.jsx

```diff
@@ -49,6 +51,21 @@ export default function SubmitButtons(props) {
         </div>
       </div>
     );
+  } else if (submission.status === 'throttledError') {
```

Comment:
The `error.type` gets translated into `submission.status`. I guess `action.field` comes in as `status` in the 1SET_SUBMISSION` reducer?

Reply:
Yeah, I'd like to refactor this. We have a Redux action that's written as a sort of generic setter action for data on the submission state. We should have actions that conceptually match what a user is doing.